windows-driver-docs-pr/devtest/28720-banned-api-usage-use-updated-function-replacement.md

```diff
@@ -9,9 +9,23 @@ f1_keywords:
 # C28720
 
 
-warning C28720: Banned API Usage
+**warning C28720: IsBadXXXPtr API Usage**\
+_Banned API Usage:  *function name* is a Banned API as listed in dontuse.h for security purposes._
```

Comment:
Is `dontuse.h` a public header? If not, we should remove this reference.

Reply:
Phil is changing the warning text, I will update this text with a new PR once that goes through!

Reply:
"Banned API Usage:  *function name* is insecure and has been marked deprecated."